unsloth/models/_utils.py

```diff
@@ -1413,3 +1415,39 @@ def validate_loftq_config(loftq_config, lora_dropout, bias, init_lora_weights, m
     pass
 
     return loftq_config
+
+def fast_inference_setup(model_name, model_config):
+    fast_inference = True
+    if not is_vLLM_available():
+        print("Unsloth: vLLM is not installed! Will use Unsloth inference!")
```

Comment:
Maybe best to ignore the print statement and change it to a warning via `UNSLOTH_LOGGING_ENABLE`